index.html

```diff
@@ -701,8 +708,107 @@
 Many of the [=principles=] described in the rest of this document extend the [=user agent=]'s duties and
 make them more precise.
 
+## Balancing Different Privacy Principles {#balancing}
+
```

Comment:
Consider "Incorporating Different Privacy Principles"

Reply:
Done. I'm not sure I exactly like this new heading, but I'd rather get this merged than wait for a perfect option.